librz/include/rz_core.h

```diff
@@ -457,6 +471,7 @@ RZ_API int rz_core_config_init(RzCore *core);
 RZ_API bool rz_core_config_eval_and_print(RzCore *core, const char *str, bool many);
 RZ_API void rz_core_config_print_all(RzConfig *cfg, const char *str, RzCmdStateOutput *state);
 RZ_API void rz_core_parse_rizinrc(RzCore *r);
+RZ_API RZ_OWN RzList *rz_core_config_variable_spaces(RZ_NONNULL RzCore *core, RZ_NULLABLE const char *space);
```

Comment:
Shall this be named `rz_core_config_in_space`?

Reply:
I don't know how to come up with a proper name

Reply:
i like ret2libc proposal.